api/src/middleware/routerTestUtils.ts

```diff
@@ -12,14 +15,26 @@ export const sessionDestroyMock = jest.fn();
 
 export const createAppAgentForRouter = (router: Router): SuperAgentTest => {
   const app = express();
+
+  const server = createServer(app);
```

Comment:
A server is created from the `app`, but `agent` is being called with `app`, not `server` below. Does this mean that the app agent won't actually handle attempted socket connections?

Does this server need to be created? Would a mock of `io` be sufficient for testing instead?

Reply:
I fixed it so that the agent is calling the server. Yes, the server will need to be created in this situation so that it can be used to test socket connections.

Reply:
Sounds good. 👍